src/ngx_http_lua_connection.c

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright (C) 2014-2015 Daurnimator
+ */
+
+/* need to include DDEBUG so that ngx_http_lua_util.h works */
+#ifndef DDEBUG
+#define DDEBUG 0
+#endif
+#include "ddebug.h"
+
+#include "ngx_http_lua_connection.h"
+
+#include <math.h> /* HUGE_VAL */
+#include <poll.h> /* POLLIN, POLLOUT */
```

Comment:
@daurnimator That's why NGINX introduced such indirections in the first place so that nginx's build system can point them to the right things on different platforms :)